apps/docs/guides/jssg/quickstart.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: "Quickstart"
+---
+
+Build a working JavaScript ast-grep (jssg) codemod in minutes.
+
+## What is jssg?
+
+JavaScript ast-grep (jssg) is a codemod framework that lets you write code transformations using JavaScript/TypeScript. Instead of complex AST manipulation, you use patterns to find and replace code structures.
+
+**How it works:**
+- Your transform runs in a **QuickJS** runtime with **AWS LLRT** module infrastructure
+- The `codemod:ast-grep` module exposes `SgRoot` and `SgNode` APIs for AST navigation
+- You write pattern-based rules to find code, then create edits to transform it
+- Return a string with changes applied, or `null`/`undefined` to skip the file
```

Comment:
It's kinda vague. We should say a jssg codemod module is a module that "export default"s a function that takes the SgRoot of a source file and returns the string of the modified source code or null/undefined...